exporter/opentelemetry-exporter-otlp-proto-common/src/opentelemetry/exporter/otlp/proto/common/log_encoder/__init__.py

```diff
@@ -22,62 +30,36 @@
     ResourceLogs,
 )
 from opentelemetry.proto.logs.v1.logs_pb2 import LogRecord as PB2LogRecord
-from opentelemetry.exporter.otlp.proto.http.trace_exporter.encoder import (
-    _encode_instrumentation_scope,
-    _encode_resource,
-    _encode_span_id,
-    _encode_trace_id,
-    _encode_value,
-    _encode_attributes,
-)
-
 
 from opentelemetry.sdk._logs import LogData
 
 
-class _ProtobufEncoder:
-    @classmethod
-    def serialize(cls, batch: Sequence[LogData]) -> str:
-        return cls.encode(batch).SerializeToString()
-
-    @staticmethod
-    def encode(batch: Sequence[LogData]) -> ExportLogsServiceRequest:
-        return ExportLogsServiceRequest(
-            resource_logs=_encode_resource_logs(batch)
-        )
+def encode_logs(batch: Sequence[LogData]) -> ExportLogsServiceRequest:
+    return ExportLogsServiceRequest(resource_logs=_encode_resource_logs(batch))
 
 
 def _encode_log(log_data: LogData) -> PB2LogRecord:
-    kwargs = {}
-    kwargs["time_unix_nano"] = log_data.log_record.timestamp
-    kwargs["span_id"] = _encode_span_id(log_data.log_record.span_id)
-    kwargs["trace_id"] = _encode_trace_id(log_data.log_record.trace_id)
-    kwargs["flags"] = int(log_data.log_record.trace_flags)
-    kwargs["body"] = _encode_value(log_data.log_record.body)
-    kwargs["severity_text"] = log_data.log_record.severity_text
-    kwargs["attributes"] = _encode_attributes(log_data.log_record.attributes)
-    kwargs["severity_number"] = log_data.log_record.severity_number.value
-
-    return PB2LogRecord(**kwargs)
+    return PB2LogRecord(
+        time_unix_nano=log_data.log_record.timestamp,
```

Comment:
Include `observed_time_unix_nano`?

Reply:
Did I remove this or is this an addition

Reply:
It's a addition and also I don't think this applies to log records emitted from SDK.